docs/sources/dashboards/use-dashboards/index.md

```diff
@@ -198,6 +198,8 @@ Click the **Refresh dashboard** icon to immediately run every query on the dashb
 
 By default, Grafana does not automatically refresh the dashboard. Queries run on their own schedule according to the panel settings. However, if you want to regularly refresh the dashboard, then click the down arrow next to the **Refresh dashboard** icon and then select a refresh interval.
 
+Selecting the "Auto" interval will schedule a refresh based on the query time range and browser window width. Short time ranges will update frequently, while longer ones will update infrequently.
```

Comment:
When you say the interval is affected by the browser window width, is that because of how much of the x-axis/time is displayed? If so, maybe that should be said explicitly.

```suggestion
Selecting the **Auto** interval schedules a refresh based on the query time range and browser window width. Short time ranges update frequently, while longer ones update infrequently.
```